src/pages/Game.tsx

```diff
@@ -13,16 +8,17 @@ import {
 } from 'react-icons/bs';
 import { FaMicrophoneAlt } from 'react-icons/fa';
 import { GiExitDoor } from 'react-icons/gi';
+import { useSelector } from 'react-redux';
 import GameComponent from '../components/GameComponent';
 import GameChat from '../components/GameChat';
 import GameWrapper from '../styles/GameStyle';
 import { Message } from '../types/AppTypes';
-import UserContext from '../context/Context';
 import { socket } from '../App';
+import { RootState } from '../store';
 
 function Game() {
+  const { username } = useSelector((state: RootState) => state.user);
```

Comment:
Same as above, but could also think about doing the thing that Sarah showed us the other day about making a reusable custom hook to streamline this, since this selector seems to be in multiple spots.